apps/frontend/client/src/lib/data/__tests__/no_global_resolver.test.ts

```
@@ -0,0 +1,75 @@
+
+    mock.module('$logger', () => ({
+      logger: { debug: () => {}, warn: () => {}, info: () => {}, error: () => {} },
+    }));
+  });
+
+  test('lpc_renderer exports createLpcRenderer instead of setLpcUrlResolver', async () => {
+    const mod = await import('../lpc_renderer');
+    // The new API
+    expect(typeof mod.createLpcRenderer).toBe('function');
+    // The old API must not exist
+    expect((mod as any).setLpcUrlResolver).toBeUndefined();
```

Biome (2.5.7)

[error] Unexpected any. Specify a different type.

(lint/suspicious/noExplicitAny)